### ISSUE TITLE
Can you share the model trained on faces?

### ISSUE THREAD

**Issue by Martyn10** [alexjc/neural-enhance#21]
Interested in testing this out but unfortunately don't have access to decent hardware to train my own.

**Comment by alexjc** [comment]
Yes. I need to retrain it so it's compatible with the current version of the code. It may take a few days as other experiments are running right now...

**Comment by Jwely** [comment]
+1

Thanks for your efforts on this, enhancing old really low resolution photos of people is definitely a use I have for this tool. Using the pre-trained model so far seems to perform worse than a simple image upscale in GIMP.

**Comment by HWiese1980** [comment]
What's the status of this issue? Is there a pre-trained model for faces for the current version of neural-enhance available? Would be great!

**Comment by alexjc** [comment]
@HWiese1980 Preparing for release v0.3 now that will address the recent bugs (e.g. that @Jwely noticed) and once it's stable I'll train on other data again. The old models for faces are not compatible with new code.

**Comment by lquessenberry** [comment]
I am interested in learning more about the high resolution faces training. What can I do to help get this updated?

**Comment by rohanpritchard** [comment]
Did this specialised model end up being shared? @alexjc Thank you! :)
